public/app/features/alerting/unified/components/silences/SilenceGrafanaRuleDrawer.tsx

```diff
@@ -12,29 +16,36 @@ type Props = {
 /**
  * For a given Grafana managed rule, renders a drawer containing silences editor and Alertmanager selection
  */
-const SilenceGrafanaRuleDrawer = ({ rulerRule, onClose }: Props) => {
-  const { uid } = rulerRule.grafana_alert;
+const SilenceGrafanaRuleDrawer = React.memo(
+  ({ rulerRule, onClose }: Props) => {
+    const { uid } = rulerRule.grafana_alert;
 
-  const formValues = getDefaultSilenceFormValues();
+    const formValues = useMemo(() => getDefaultSilenceFormValues(), []);
```

Comment:
Do we need this at all? `SilencesEditor` populates `formValues` by default

Reply:
you're right, SilenceEditor is rendered also by NewSilencePage where it uses another default values, but in this case we can skip the property as we have the default.

Reply:
commit pushed